src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```
@@ -126,7 +124,14 @@ public void SetDataSource(string dataSourceFileName)
 
       public void ImportData(object sender, EventArgs e)
       {
-         OnTriggerImport.Invoke(this, new ImportTriggeredEventArgs { DataSource = _dataSource });
+         try
+         {
+            OnTriggerImport.Invoke(this, new ImportTriggeredEventArgs { DataSource = _dataSource });
+         }
+         catch (Exception exception)
```

Catch all Exception? why?

---

Fixed